examples/twoadicity/src/main.leo

```diff
@@ -0,0 +1,27 @@
+// The 'twoadicity' main function.
+@program
+function main(public n: field) -> u8 {
+  let remaining_n: field = n;
+  let powers_of_two: u8 = 0u8;
+  // Since field ints are 253 bits or fewer,
+  // any number in the field will have at most 252 powers of two in its prime factoring.
+  for i:u8 in 0u8..252u8 {
+    if is_even_and_nonzero(remaining_n) {
+      remaining_n = remaining_n / 2field;
+      powers_of_two = powers_of_two + 1u8;
+      }
+  }
+  return powers_of_two;
+}
+
+/* We don't yet have shifts or bitmasks or % for fields,
+   but we can define evenp as in the definition below.
```

Comment:
```suggestion
/* We define evenp as in the definition below.
```

Reply:
I reworded the comment a little differently, I hope better.